src/website/app/demos/Card/examples/card-image/withImage.js

```diff
@@ -21,7 +21,7 @@ import DemoLayout from '../../components/DemoLayout';
 
 export default {
   id: 'with-image',
-  title: 'Example',
+  title: 'Displaying Images in a Card',
```

Comment:
It's doubtful they'd need to display more than one image in a Card. Should this be "Displaying an Image in a Card" instead?

Reply:
👍